tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -925,6 +965,35 @@ def remove_tmp_onnx_model(out_dir):
             os.remove(tmp_onnx_model)
 
 
+def convert_pytorch_via_onnx(args, example_inputs, cli_parser, framework):
+    opset_version = None
+    if 'onnx_opset_version' in args and args['onnx_opset_version'] is not None:
+        opset_version = args['onnx_opset_version']
+
+    out_dir = args['output_dir'] if 'output_dir' in args else None
+
+    model_onnx = convert_pytorch_to_onnx(args['input_model'],
+                                         parse_input_shapes(args),
+                                         opset_version,
+                                         example_inputs,
+                                         out_dir)
+
+    args['input_model'] = model_onnx
+    if os.environ.get('SAVE_TO_BYTES_IO_ONNX_MODEL'):
+        args['use_legacy_frontend'] = True
+    args['example_input'] = None
+    args['onnx_opset_version'] = None
+
+    try:
```

Comment:
I think we need to move a call of `convert_pytorch_to_onnx` also under `try`. Just to avoid remaining temporal onnx file in case of failures of `convert_pytorch_to_onnx`, for example.